### PR TITLE
Remove dead/unused code

### PR DESCRIPTION
**Pull request by mvandenburgh**
Based on #192 and should be merged after that PR.

- Removes a `dataclass` that is no longer used
- Remove the server status endpoint - this isn't used anywhere.

### CODE REVIEW

**Review by BryonLewis — CHANGES_REQUESTED** [review]
I'm using the server status to check to see that the container is healthy so I can start the celery container.  We either need to remove the migration from the entrypoint (which may be a good idea to prevent bad migrations).  Or find another way to prevent both the celery and django container in the dev environment from attempting to run migrations in parallel (something that can cause issues)

**Comment by mvandenburgh** [comment]
> I'm using the server status to check to see that the container is healthy so I can start the celery container. We either need to remove the migration from the entrypoint (which may be a good idea to prevent bad migrations). Or find another way to prevent both the celery and django container in the dev environment from attempting to run migrations in parallel (something that can cause issues)

ok. For now I dropped that change, but I agree we should find a way to handle migrations better

File: django/src/rdwatch/dataclasses/server_status.py [hunk 1]
```diff
@@ -1,9 +0,0 @@
-from dataclasses import dataclass
-from datetime import timedelta
-
-
-@dataclass
-class ServerStatus:
-    uptime: timedelta
-    hostname: str
-    ip: str
```
Comment (BryonLewis) [inline django/src/rdwatch/dataclasses/server_status.py:9]: I think that ./rdwatch/serializers/server_status.py needs this as an import?

**Review by BryonLewis — CHANGES_REQUESTED** [review]

**Review by BryonLewis — APPROVED** [review]
pulled, tested, and LGTM. :+1:

**Pull request merged**